src/components/Tabs.tsx

```diff
@@ -157,10 +153,12 @@ export function Tabs<V extends string>(props: TabsProps<V, {}> | RouteTabsProps<
     }
   }
 
-  // We also check this in TabsWithContent, but if someone is using Tabs standalone, check it here as well
+  const actionsDiv = <div css={Css.ml("auto").addIn("&>div", Css.df.aic.childGap1.$).$} ref={tabActionsRef} />;
+
+  // Check to see if we should hide the tabs. We will still return something so that the Tab Actions (if any) can render
   const hideTabs = props.alwaysShowAllTabs ? false : (props.tabs as any[]).filter((t) => !t.disabled).length === 1;
   if (hideTabs) {
-    return <></>;
+    return <div css={Css.df.aic.$}>{actionsDiv}</div>;
```

Comment:
I was thinking maybe we could sometimes leave this out, but I guess at this point we're not sure if a child will/will not end up using `TabActions` so we need to always include it. Makes sense.